index.html

```diff
@@ -27,6 +27,83 @@ <h2>I’m a software developer</h2>
         <li><a href="#"><img src="angellist.png" alt="angellist"></a></li>    
     </ul>  
     </section>
+
+    
+    <header class="title">
+        <div><a href="title"><img src="Disabled.png" alt="scroll down button"></a></div>
+        <div><h1>My recent Work</h1></div>
+        <div><img src="indicator.png" alt="indicator"></div>
+    </header>
+    <section class="descript">
```

Comment:
- [ ] Nice working on your "Works" Section, Kindly make it a 1Grid layout, so it aligns with what you have in the Figma Template
- [ ] As part of the project requirement, kindly add the About Me section, just as you have it in the Figma Design.

### Your Design
![yr-2](https://user-images.githubusercontent.com/48371156/170841105-c7d43465-f22a-4937-be9d-996fb5ba96b4.png)

### Figma Design
![yr-1](https://user-images.githubusercontent.com/48371156/170841109-e104deb5-db69-4b9e-87c1-2e12e54a520e.png)

### About Me
![abt_me](https://user-images.githubusercontent.com/48371156/170841255-b2250ef5-22bf-42ce-af6e-8ef580f54a82.png)

Reply:
Thanks for your review, I will have in mind your advice. Regards.